apps/etterlatte-saksbehandling-ui/server/src/routers/loggerRouter.ts

```diff
@@ -34,29 +62,20 @@ loggerRouter.post('/', express.json(), (req, res) => {
               stack_trace: `Error occurred in ${component}:\n${message}\n${error}`,
               user_device: JSON.stringify(body.jsonContent.userDeviceInfo),
               user_agent: body.jsonContent.userAgent,
+              url: maybeUrl,
             })
           })
           .catch((err) => {
-            frontendLogger.error('Request got error on: \n', err)
+            frontendLogger.error('Request got error on: \n', err, '\n url: ', maybeUrl)
           })
       }
     } else {
       frontendLogger.error(
-        `General error from frontend: ${JSON.stringify(body.data)} \n details: ${JSON.stringify(body.jsonContent)}`
+        `General error from frontend: ${JSON.stringify(body.data)} \n details: ${JSON.stringify(
+          body.jsonContent
+        )} url: ${maybeUrl}`
```

Comment:
Ville kanskje prøvd å bygge opp et objekt for å gjøre det litt mer lesbart i kibana. 

Eks.: 

```
      frontendLogger.error({
        ...mapData(body.data),
        ...mapJsonContent(body.jsonContent),
        request_uri: maybeUrl,
      })
```


``` 
function mapData(data: any): object {
  try {
    const d = JSON.parse(data)
    return {
      message: d.msg,
      error_info: d.errorInfo || undefined,
    }
  } catch (_) {
    return {
      message: 'Ukjent feil oppsto:',
      error_data: JSON.stringify(data),
    }
  }
}

function mapJsonContent(jsonContent: any): object {
  try {
    return {
      location: jsonContent.url,
      user_device: JSON.stringify(jsonContent.userDeviceInfo),
      user_agent: jsonContent.userAgent,
    }
  } catch (_) {
    return {
      json_content: JSON.stringify(jsonContent),
    }
  }
}
```

Reply:
Bare basert på error logger i Kibana. Aner ikke om det faktisk fungerer... 🤷‍♂️